CompressImagesFunction/CompressImagesFunction.cs

```diff
@@ -16,17 +16,19 @@ public static class CompressImagesFunction
         public static Task Trigger(
             [QueueTrigger("compressimagesmessage")]CompressImagesMessage compressImagesMessage,
             [Queue("openprmessage")] ICollector<OpenPrMessage> openPrMessages,
+            IRepoChecks repoChecks,
```

Comment:
ahhh. This will actually break the function from running. We don't have any injection like this setup.

We have to new it up ourselves in here

Reply:
Oops, sorry!

Reply:
No worries. It's no big deal :)